### DISCUSSION TITLE
ESLint not working for `vscode-langservers-extracted@>=4.9.0`

### DISCUSSION THREAD

**Discussion by lizclipse** [helix-editor/helix#11506] [Troubleshooting] [answered]
A while ago I spent ages trying to get ESLint working, and eventually I did, but at some point in the last few months it just stopped again and I haven't been able to figure out what changed until today. At some point I upgraded `vscode-langservers-extracted` to 4.10.0 and since then it hasn't worked, but I've only found this by downgrading to to 4.8.0 and suddenly I have lints back in helix.

I'm making this a discussion mostly because it's not a helix issue, but I think it's worth putting here in case someone else is having the same issue. The main problem with trying to diagnose this is that the LSP _is_ running, but it just doesn't output any diagnostics, so trying to search what could be the problem is nearly impossible. I am going to try exploring what the root issue is, but if anyone has hit this before and knows more than me let me know, it would be very handy.

**Comment by David-Else** [comment, marked as answer]
I think versions higher than n`4.8.0` need https://github.com/helix-editor/helix/pull/11315 as ESLint langserver had a big update , this is based on some memory I have from a discussion on GitHub somewhere, maybe give it a try.

**Reply by lizclipse** [reply]
This was it, thank you. I've tested the PR against 4.10.0 and it works unlike master, so this issue should be resolved once that's in.

**Comment by babucarr32** [comment]
Using a previous version `4.8.0` worked for me. Specifically `npm i -g vscode-langservers-extracted@4.8.0`
